wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Commands.java

```diff
@@ -166,6 +166,19 @@ public static Command either(Command onTrue, Command onFalse, BooleanSupplier se
     return new ConditionalCommand(onTrue, onFalse, selector);
   }
 
+  /**
+   * Runs a command if the boolean condition function is true otherwise it does nothing and exits.
+   *
+   * @param condition the condition function
+   * @param onTrue the command to run if the condition function returns true
+   * @return the command
+   * @see ConditionalCommand
+   * @see {@link Commands#none()}
```

Comment:
```suggestion
   * @see Commands#none()
```
When running the java formatting CI locally, it didn't seem to like using `@link` in the `@see`. This has the following javadoc output (you can run `./gradlew wpilibNewCommands:javadoc` locally and open `wpilibNewCommands/build/docs/javadoc/allclasses-index.html` to see the javadoc output):
<img width="1286" alt="image" src="https://github.com/user-attachments/assets/7e89fc66-7048-4053-a2fc-8ace8458f109" />
(This is including the other suggested change)